server/src/repositories/event.repository.ts

```diff
@@ -25,14 +24,8 @@ import { ImmichLogger } from 'src/utils/logger';
   path: '/api/socket.io',
   transports: ['websocket'],
 })
-export class CommunicationRepository
-  implements OnGatewayConnection, OnGatewayDisconnect, OnGatewayInit, ICommunicationRepository
-{
-  private logger = new ImmichLogger(CommunicationRepository.name);
```

Comment:
This is all basically gone - no more need to store, register, and invoke callbacks.